guides/back-end-projects.md

```diff
@@ -0,0 +1,104 @@
+---
+explains: The style guide to the way we organize our back-end projects
+---
+
+# How we use docker and docker-compose
+
+A lot of the software we make at City of Amsterdam uses docker containers.
```

Comment:
only data.amsterdam.nl software. There is 10 times more non containerized software around I'm afraid within the municipality.